src/patternfly/components/Timestamp/timestamp.scss

```diff
@@ -0,0 +1,43 @@
+.pf-c-timestamp {
+  // Timestamp text
+  --pf-c-timestamp__text--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-timestamp__text--Color: var(--pf-global--Color--200);
+
+  // Help text variables for the timestamp text
+  --pf-c-timestamp__text--m-help-text--TextDecorationLine: underline;
```

Comment:
For the var syntax, I had followed how Progress Stepper and Description List had them setup:

```
--pf-c-description-list__text--m-help-text--cssProperty
--pf-c-progress-stepper__step-title--m-help-text--cssProperty
```

If the `baseComponent--m-help-text__subComponent` syntax would still be better to use, would you want me to add a comment to the progress stepper and description list scss files stating to update them at some point?

Reply:
Progress stepper and description list help text mod applies to the the text and title elements, whereas `pf-m-help-text` in timestamp applies to the root component rather than `.pf-c-timestamp__text`. I assume the reason for nested elements is that the help text modifier styling applies to all children, including icons etc., otherwise we would use a layout for icon + timestamp.

Reply:
Ah gotcha, that makes sense! Thanks for the clarification, will add that update as well